src/sql/workbench/contrib/editData/browser/editDataGridPanel.ts

```diff
@@ -1161,6 +1190,7 @@ export class EditDataGridPanel extends GridParentComponent {
 		}
 		else if (Services.DBCellValue.isDBCellValue(value)) {
 			valueToDisplay = (value.displayValue + '');
+			valueToDisplay = valueToDisplay.replace(/(\r\n|\n|\r)/g, '\u0000');
```

Comment:
is this really needed? line breaks won't be displayed anyway, right?

Reply:
They will actually, if you don't do this check, this will happen:

![image](https://user-images.githubusercontent.com/18018452/224176987-4c11156c-42c1-4782-b27d-229ba7f29b05.png)

It doesn't match what is shown in SSMS, so yes it is needed.